tasking/pulp/tasking/base.py

```diff
@@ -1,20 +1,22 @@
-from contextlib import suppress
-from gettext import gettext as _
 import logging
 import time
 import traceback
 import uuid
+from contextlib import suppress
+from gettext import gettext as _
 
-from celery import task, Task as CeleryTask, current_task
+from celery import Task as CeleryTask
+from celery import current_task, task
 from celery.app import control
 from celery.result import AsyncResult
 from django.db.models import Count
 
-from pulp.app.models import Worker, ReservedResource, Task as TaskStatus
-from pulp.common import TASK_FINAL_STATES, TASK_INCOMPLETE_STATES, TASK_STATES
-from pulp.exceptions import PulpException, MissingResource, PulpCodedException
-from pulp.tasking.celery_instance import celery, RESOURCE_MANAGER_QUEUE, DEDICATED_QUEUE_EXCHANGE
-
+from pulp.app.models import Task as TaskStatus
+from pulp.app.models import ReservedResource, TaskLock, Worker
+from pulp.common import (
+    RESOURCE_MANAGER_WORKER_NAME, TASK_FINAL_STATES, TASK_INCOMPLETE_STATES, TASK_STATES)
+from pulp.exceptions import MissingResource, PulpCodedException
+from pulp.tasking.celery_instance import DEDICATED_QUEUE_EXCHANGE, RESOURCE_MANAGER_QUEUE, celery
```

Comment:
I recommend making it two lines:
This is really a matter of opinion. I'm OK with whatever you decided to do here. My recommendation is:

```
from pulp.tasking.celery_instance import celery
from pulp.tasking.celery_instance import DEDICATED_QUEUE_EXCHANGE, RESOURCE_MANAGER_QUEUE
```